types/underscore/underscore-tests.ts

```diff
@@ -1003,60 +1003,40 @@ declare const extractChainTypes: ChainTypeExtractor;
     _(stringRecordDictionary).find(stringRecordProperty); // $ExpectType StringRecordOrUndefined
     extractChainTypes(_.chain(stringRecordDictionary).find(stringRecordProperty)); // $ExpectType ChainType<StringRecordOrUndefined, never>
 
-    // property name iteratee - dictionaries - detect
-    _.detect(stringRecordDictionary, stringRecordProperty); // $ExpectType StringRecordOrUndefined
-    _(stringRecordDictionary).detect(stringRecordProperty); // $ExpectType StringRecordOrUndefined
-    extractChainTypes(_.chain(stringRecordDictionary).detect(stringRecordProperty)); // $ExpectType ChainType<StringRecordOrUndefined, never>
+    // property name iteratee - lists - detect
+    _.detect(stringRecordList, stringRecordProperty); // $ExpectType StringRecordOrUndefined
+    _(stringRecordList).detect(stringRecordProperty); // $ExpectType StringRecordOrUndefined
+    extractChainTypes(_.chain(stringRecordList).detect(stringRecordProperty)); // $ExpectType ChainType<StringRecordOrUndefined, never>
 
     // property path iteratee - lists - find
     _.find(stringRecordList, stringRecordPropertyPath); // $ExpectType StringRecordOrUndefined
     _(stringRecordList).find(stringRecordPropertyPath); // $ExpectType StringRecordOrUndefined
     extractChainTypes(_.chain(stringRecordList).find(stringRecordPropertyPath)); // $ExpectType ChainType<StringRecordOrUndefined, never>
 
-    // property path iteratee - lists - detect
-    _.detect(stringRecordList, stringRecordPropertyPath); // $ExpectType StringRecordOrUndefined
-    _(stringRecordList).detect(stringRecordPropertyPath); // $ExpectType StringRecordOrUndefined
-    extractChainTypes(_.chain(stringRecordList).detect(stringRecordPropertyPath)); // $ExpectType ChainType<StringRecordOrUndefined, never>
-
-    // property path iteratee - dictionaries - find
-    _.find(stringRecordDictionary, stringRecordPropertyPath); // $ExpectType StringRecordOrUndefined
-    _(stringRecordDictionary).find(stringRecordPropertyPath); // $ExpectType StringRecordOrUndefined
-    extractChainTypes(_.chain(stringRecordDictionary).find(stringRecordPropertyPath)); // $ExpectType ChainType<StringRecordOrUndefined, never>
-
     // property path iteratee - dictionaries - detect
     _.detect(stringRecordDictionary, stringRecordPropertyPath); // $ExpectType StringRecordOrUndefined
     _(stringRecordDictionary).detect(stringRecordPropertyPath); // $ExpectType StringRecordOrUndefined
     extractChainTypes(_.chain(stringRecordDictionary).detect(stringRecordPropertyPath)); // $ExpectType ChainType<StringRecordOrUndefined, never>
+
+    // identity iteratee - dictionaries - find
+    _.find(stringRecordDictionary, stringRecordProperty); // $ExpectType StringRecordOrUndefined
+    _(stringRecordDictionary).find(stringRecordProperty); // $ExpectType StringRecordOrUndefined
+    extractChainTypes(_.chain(stringRecordDictionary).find(stringRecordProperty)); // $ExpectType ChainType<StringRecordOrUndefined, never>
```

Comment:
There are not identity iteratee tests, are they?

Reply:
You're totally right, thanks for the catch here and below!